frontend/javascript/components/document-collection.vue

```diff
@@ -193,29 +206,39 @@ export default {
     }
   },
   data() {
+    const queryParams = new URLSearchParams(window.location.search)
+
     const documents = []
     const directories = []
     let collection = {
       documents,
       directories
     }
+    let directoryStack = []
+    let currentDirectory = null
+    let document = null
     if (this.documentCollection) {
       collection = this.documentCollection
+      directoryStack = collection.directory_stack
+      currentDirectory = collection.current_directory
+      document = collection.documents.find(
```

Comment:
what if the document is at the bottom of the view and not yet loaded?

Reply:
resolved by [`746d9fd`](https://github.com/okfde/django-filingcabinet/pull/78/commits/746d9fd8376a2d9343efc61778e061c14e20b262)